src/utils/graphql/queries/getElements.query.ts

```diff
@@ -0,0 +1,17 @@
+import { gql } from '@apollo/client'
+
+export default gql`
+  query getElements($sectionId: Int!) {
+    templateElements(filter: { sectionId: { equalTo: $sectionId } }) {
```

Comment:
Just wondering why you used `filter` here rather than `condition`. (Filter is from advanced filtering plugin, condition is built-in). I guess it doesn't really matter, but there is a difference when using live queries (not really an issue here). 

Do you think we should stick to using `condition` where we can and only extend to the advanced `filter` when we really need it?

Reply:
Yes, keeping consistency will be be better. I haven't seen the difference and it works the same with `condition`. I'll change this now.